config/rbac/role.yaml

```diff
@@ -4,176 +4,274 @@ kind: ClusterRole
+      - watch
+  - apiGroups:
+      - apps
+    resources:
+      - daemonsets
+      - deployments
+      - replicasets
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - networking.k8s.io
+    resources:
+      - networkpolicies
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - operator.kyma-project.io
+    resources:
+      - telemetries
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - rbac.authorization.k8s.io
+    resources:
+      - clusterrolebindings
+      - clusterroles
+      - rolebindings
+      - roles
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - security.istio.io
+    resources:
+      - peerauthentications
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - telemetry.kyma-project.io
+    resources:
+      - logparsers
+      - logpipelines
+      - metricpipelines
+      - tracepipelines
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
+      - telemetry.kyma-project.io
+    resources:
+      - logparsers/finalizers
+      - logpipelines/finalizers
+      - metricpipelines/finalizers
+    verbs:
+      - update
+  - apiGroups:
+      - telemetry.kyma-project.io
+    resources:
+      - logparsers/status
+      - logpipelines/status
+      - metricpipelines/status
+      - tracepipelines/status
+    verbs:
+      - get
+      - patch
+      - update
+
+  # policy rules for fluent bit
+  - apiGroups:
+      - ""
+    resources:
+      - namespaces
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
+
+  # policy rules for metric agent
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+      - nodes/metrics
+      - nodes/stats
+      - nodes/proxy
+      - nodes/spec
+      - services
+      - endpoints
+      - pods
+      - pods/status
+      - events
+      - namespaces
+      - namespaces/status
+      - replicationcontrollers
+      - replicationcontrollers/status
+      - resourcequotas
+    verbs:
+      - get
+      - list
+      - watch
+  - nonResourceURLs:
+      - /metrics
+      - /metrics/cadvisor
+    verbs:
+      - get
+  - apiGroups:
+      - apps
+    resources:
+      - daemonsets
+      - deployments
+      - replicasets
+      - statefulsets
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - extensions
+    resources:
+      - daemonsets
+      - deployments
+      - replicasets
+      - statefulsets
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - batch
+    resources:
+      - jobs
+      - cronjobs
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - autoscaling
+    resources:
+      - horizontalpodautoscalers
+    verbs:
+      - get
+      - list
+      - watch
+
+  # policy rules for metric gateway
```

Comment:
#########################
 # Policy rules for metric-gateway
 #########################